subprojects/testing-native/src/integTest/groovy/org/gradle/nativeplatform/test/xctest/SwiftXCTestIntegrationTest.groovy

```diff
@@ -212,9 +205,9 @@ apply plugin: 'swift-library'
     // TODO: Need to support test report for test case assertion
     @Requires(TestPrecondition.MAC_OS_X)
     def "executes added test suite and case"() {
-        def testBundle = new IncrementalSwiftXCTestAddDiscoveryBundle()
-
         given:
+        def testBundle = new IncrementalSwiftXCTestAddDiscoveryBundle()
+        settingsFile << "rootProject.name = 'app'"
```

Comment:
This should probably `${testBundle.projectName}`.

Reply:
Same as above